extension/modules/personas.js

```diff
@@ -1123,8 +1123,8 @@ var PersonaController = {
         // Update the Custom menu. Custom personas unavailable in per-window
         // personas mode.
         let customMenu = document.getElementById("personas-plus-custom-menu");
-        customMenu.hidden = this._prefs.get("perwindow") || !this._prefs.get("showCustomMenu");
-        if (!customMenu.hidden) {
+        customMenu.disabled = this._prefs.get("perwindow")
```

Comment:
`customMenu.hidden` is better, I suppose. I created a Custom Persona with name, toggled `perwindow` pref to `true` and checked toolbar menu. Custom menu displays as blank.